themes/openy_themes/openy_carnation/templates/node/node--location-type--teaser.html.twig

```diff
@@ -111,7 +111,7 @@
     {{ content.field_location_address }}
 
     {% if directions_title %}
```

Comment:
I'd replace it with {{ content.field_location_directions }} and get rid of all the excessive code that adds the `direction_*` variables